cmd/osv-scanner/main.go

```diff
@@ -124,9 +124,13 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Name:  "experimental-all-packages",
 				Usage: "when json output is selected, prints all packages",
 			},
+			&cli.BoolFlag{
+				Name:  "experimental-licenses-summary",
+				Usage: "report a license summary, implying the --experimental-all-packages flag",
```

Comment:
do you think this is worth documenting in the usage message or do you think it clutters it? I'm not sure if the user would find this behaviour unexpected.

Reply:
@another-rex

Reply:
I think the wording here is fine, makes it clearer, and we can always remove it later if needed.